cis_enforcements/aws/cis_v300_s1_iam/support_role.tf

```diff
@@ -0,0 +1,54 @@
+# AWS > IAM > Stack
+resource "turbot_policy_setting" "aws_iam_stack" {
+  resource = turbot_smart_folder.aws_cis_v300_s1_iam.id
+  type     = "tmod:@turbot/aws-iam#/policy/types/iamStack"
+  note     = "AWS CIS v3.0.0 - Controls:  1.17"
+  value    = "Check: Configured"
+  # value    = "Enforce: Configured"
+
+}
+
+# AWS > IAM > Stack > Source
+resource "turbot_policy_setting" "aws_iam_stack_source" {
+  resource       = turbot_smart_folder.aws_cis_v300_s1_iam.id
+  type           = "tmod:@turbot/aws-iam#/policy/types/iamStackSource"
+  note           = "AWS CIS v3.0.0 - Controls:  1.17"
+  template_input = <<EOT
+  {
+    account  {
+      Id
+      metadata
+    }
+  }
+  EOT
```

Comment:
Please checkout the link in the design spec about using **_Indented_ HEREDOC** format.

should be <<-EOT  and the lines should be indented 2 spaces more.